### ISSUE TITLE
Saving while modded tab is active causes crash if mod is removed

### ISSUE THREAD

**Issue by southpawgeek** [PonyWarrior/Hades2ModRepo#3]
Saving the game while the PonyMenu tab is active, then loading it without PonyMenu installed will cause it to crash as soon as you open the inventory. This is an issue for people who either remove the mod, or play on multiple machines that aren't all modded.

Specifically, it'll break on ResourceLogic.lua line 341. 

To reproduce:
1. open inventory and view PonyMenu, leaving it as the last viewed tab
2. make game save (change rooms)
3. quit
4. uninstall PonyMenu
5. log in and try to open inventory

**Comment by PonyWarrior** [comment]
Well I could make a fix that prevents that, however then it gets very annoying to open your inventory and have to switch to a different tab. It's simpler for users to just close on a vanilla tab before uninstalling.

**Comment by southpawgeek** [comment]
I agree that it'd be very annoying to re-select the tab every time you close inventory, especially if you're trying to apply multiple boons. 

The main problem I foresee is most users won't know what's happening and would blame the mod/tools. Maybe consider adding a small warning to the uninstall instructions?

**Comment by PonyWarrior** [comment]
I have done so after seeing your issue here.

**Comment by southpawgeek** [comment]
Sounds good!
